BraveShared/BraveUX.swift

```diff
@@ -8,6 +8,7 @@ public struct BraveUX {
     public static let braveOffersURL = URL(string: "https://offers.brave.com/")!
     public static let braveRewardsLearnMoreURL = URL(string: "https://brave.com/rewards-ios/")!
     public static let braveRewardsUnverifiedPublisherLearnMoreURL = URL(string: "https://brave.com/faq-rewards/#unclaimed-funds")!
+    public static let braveTodayPartnersURL = URL(string: "https://brave.com/brave-today/")!
```

Comment:
Is this url final? shouldn't we be taken to a support page of some kind

Reply:
Yeah this URL is final, got it from Matt